puppet/preprovision.sh

```diff
@@ -38,8 +38,10 @@ if [[ ! -f /etc/chassis-updated ]]; then
 	# Update apt
 	sudo apt-get update
 
-	# Install/Upgrade Puppet
-	sudo apt-get -q -y install puppet
+	if ps aux | grep "puppet agent" | grep -v grep 2> /dev/null
```

Comment:
I think using `which puppet` for a check instead of `ps aux` would be better as puppet is probably not running yet.

Reply:
I pushed up removal of this conditional, I think it's best as a future optimisation, eitherway it didn't like what I had here

Reply:
Awesome. Thanks for that!